integrated_channels/moodle/client.py

```diff
@@ -131,7 +132,11 @@ def __init__(self, enterprise_configuration):
         """
         super().__init__(enterprise_configuration)
         self.config = apps.get_app_config('moodle')
-        self.token = enterprise_configuration.token or self._get_access_token()
+        self.token = (
+            enterprise_configuration.decrypted_token
+            if getattr(settings, 'FEATURES', {}).get('USE_ENCRYPTED_USER_DATA', False)
+            else enterprise_configuration.token
+        ) or self._get_access_token()
```

Comment:
This can be simply written as:

```python

token = enterprise_configuration.decrypted_token if settings.FEATURES.get("USE_ENCRYPTED_USER_DATA", False) \
            else enterprise_configuration.token
self.token = token or self._get_access_token()

Reply:
@sameenfatima78 
"getattr(settings, 'FEATURES', {})"
this was used for test run where settings did not have access to FEATURES.

Reply:
Okay, but we can still simplify the above statement as mentioned above

Reply:
@sameenfatima78 for simplification i have formatted it.